data-requirements/summary-compare.ts

```diff
@@ -179,13 +197,14 @@ async function main() {
       elmParserDRByKey.forEach(dr => {
         const elmParserMustSupports = dr.mustSupport;
         const elmParserValueSet = dr.codeFilter?.find(cf => cf.valueSet)?.valueSet;
```

Comment:
```suggestion
        // ignore valueset codefilter if it's a "value" type, TODO: get rid of explicit value check and
        // use primary code path from proposed extension added to dr, once implemented/available
        const elmParserValueSet = dr.codeFilter?.find(cf => cf.valueSet && cf.path !== 'value')?.valueSet;
```

Reply:
Sorry, for some reason I thought I added this but I guess I didn't!